rules/hand-over/rule.md

```diff
@@ -117,9 +121,9 @@ Done - As per our meeting the handover has been completed to my satisfaction
 :::
 :::
 ::: good
-Figure: Good example - This handover has lots of URLs and is complete
+Figure: Good example - This handover has a clear designation of responsibilities, lots of URLs and is complete
 :::
 
-If you need to handover only a single task there are more details on [Do you know how to hand over email tasks to others?](/how-to-hand-over-tasks-to-others)
+If you need to hand over only a single task there are more details on [Do you know how to hand over email tasks to others?](/how-to-hand-over-tasks-to-others)
```

Comment:
Should this be a single word? from `hand over` to `handover`

Reply:
Hand over = verb
Handover = noun